lib/rake/extensiontask.rb

```diff
@@ -377,9 +377,6 @@ def define_cross_platform_tasks(for_platform)
           @lib_dir = "#{@lib_dir}/#{$1}"
         end
 
-        # Update cross compiled platform/version combinations
-        @ruby_versions_per_platform[for_platform] << version
```

Comment:
This line was added in #166 by you.
Could you confirm that this doesn't break #166 case?

Reply:
Originally it was added in #137, but modified later on and it was actually obsoleted by #166, but I didn't notice that.
I can confirm that both use cases (#137 and #166) work properly.

Reply:
Thanks.